packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -802,6 +828,10 @@ export class SankeyChartBase extends React.Component<ISankeyChartProps, ISankeyC
   }
 
   public render(): React.ReactNode {
+    const formatNumber = memoizeFunction((formatter?: NumberFormatter) => {
```

Comment:
you can define this const outside of render and use in all places to format instead of passing through all functions

Reply:
updated. created private method `_formatNumber` and used it in other places